README.md

```diff
@@ -1,24 +1,27 @@
 Leaflet.functionaltilelayer
 ===========================
 
+For use with Leaflet 1.0.1.
```

Comment:
I will add a version number 1.0.0 to package.json. Maybe you could create a tag for 0.7.